src/test/java/com/sonymobile/tools/gerrit/gerritevents/SSHUtilTest.java

```diff
@@ -24,9 +23,7 @@
     @Test
     public void testNoPassphraseParsing() throws Exception {
         // Get no-passphrase key resource as file
-        URL url = Thread.currentThread().getContextClassLoader().getResource(
-                "com/sonymobile/tools/gerrit/gerritevents/id_rsa");
-        File file = new File(url.getPath());
+        File file = new File("src/test/resources/com/sonymobile/tools/gerrit/gerritevents/id_rsa");
```

Comment:
Why did you change this?
this won't work when running the tests from the published test jar

Reply:
ok, but with the url it doesn't work with maven (at least on windows)

Reply:
Yea, I would guess this is full of test code not working on windows :) Though still a bit strange why this wouldn't work.